docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.7'
+
+services:
```

Comment:
Add please `container_name` for each container to make it easier to interact with them.